soc/xlnx/versalnet/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_AMD_VERSALNET_RPU
```

Comment:
add 'select ARM_MPU' here , then you can remove the other instances of 'CONFIG_ARM_MPU=y'.

Reply:
okay will fix